railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults.rb.tt

```diff
@@ -1,37 +1,36 @@
 # Be sure to restart your server when you modify this file.
-# This file contains all the new default configuration options from
-# Rails 5.0.
-<%- unless options[:skip_active_record] -%>
-
-# Require `belongs_to` associations by default. This is a new Rails 5.0
-# default, so it is introduced as a configuration option to ensure that apps
-# made on earlier versions of Rails are not affected when upgrading.
-Rails.application.config.active_record.belongs_to_required_by_default = true
-<%- end -%>
-
-# Do not halt callback chains when a callback returns false. This is a new
-# Rails 5.0 default, so it is introduced as a configuration option to ensure
-# that apps made with earlier versions of Rails are not affected when upgrading.
-ActiveSupport.halt_callback_chains_on_return_false = false
+#
+# This file contains migration options to ease your Rails 5.0 upgrade.
+# They will be removed in the next major Rails version.
+#
+# Flip the defaults to the value of past versions then once upgraded
+# begin migrating to the new default one by one.
+#
+# Read the Rails 5.0 release notes for more info on each option.
 <%- unless options[:api] -%>
 
-# Enable per-form CSRF tokens.
+# Enable per-form CSRF tokens. This is new feature of Rails 5.
 Rails.application.config.action_controller.per_form_csrf_tokens = true
 
-# Enable origin-checking CSRF mitigation.
+# Enable origin-checking CSRF mitigation. This is new feature of Rails 5.
 Rails.application.config.action_controller.forgery_protection_origin_check = true
 <%- end -%>
 
-# Configure SSL options to enable HSTS with subdomains. This is a new
-# Rails 5.0 default, so it is introduced as a configuration option to ensure
-# that apps made on earlier versions of Rails are not affected when upgrading.
-Rails.application.config.ssl_options = { hsts: { subdomains: true } }
-
-# Preserve the timezone of the receiver when calling to `to_time`.
-# Ruby 2.4 will change the behavior of `to_time` to preserve the timezone
-# when converting to an instance of `Time` instead of the previous behavior
-# of converting to the local system timezone.
-#
-# Rails 5.0 introduced this config option so that apps made with earlier
-# versions of Rails are not affected when upgrading.
+# Make Ruby 2.4 preserve the timezone of the receiver when calling `to_time`.
 ActiveSupport.to_time_preserves_timezone = true
+
+# Following config options are introduced to ensure that apps
+# made on earlier versions of Rails are not affected when upgrading.
```

Comment:
Think this isn't true as the defaults introduced with new features in Rails 5 have ramifications on upgrading apps. Let's just remove this because the whole file has become about ensuring this 😁